lib/tasks/whitehall_importer.rb

```diff
@@ -145,16 +145,26 @@ def create_edition(document, translation, whitehall_edition, edition_number)
     end
 
     def initial_status(whitehall_edition, revision)
+      author = if set_published_state_first?(whitehall_edition["state"])
+                 published_author(whitehall_edition)
+               else
+                 whitehall_edition["revision_history"].last
```

Comment:
I've done the same in my unpublishing PR using the last revision_history entry to identify the user who created the status, however this could be inaccurate as a document can be edited after it's had it's state changed by a different publisher. Not sure how much it matters though!

Reply:
Hmmm. That's a good point. I'll change this get the `revision_history` for the edition state.